prowler/providers/aws/services/trustedadvisor/trustedadvisor_premium_support_plan_subscribed/trustedadvisor_premium_support_plan_subscribed.metadata.json

```diff
@@ -3,7 +3,7 @@
   "CheckID": "trustedadvisor_premium_support_plan_subscribed",
   "CheckTitle": "Check if a Premium support plan is subscribed",
   "CheckType": [],
-  "ServiceName": "support",
+  "ServiceName": "trustedadvisor",
```

Comment:
We need to review this because the service is `trustedadvisor`.

Reply:
LGTM